content/tutorials/history.textile

```diff
@@ -375,7 +453,25 @@ In this demo we use the "history API":/realtime/history to replay up to the last
 h2. Download tutorial source code
 
 blang[java].
-  The complete source code for each step of "this tutorial is available on Github":https://github.com/piotrekinventica/tutorials/commits/history-java-rest.
+  The complete source code for each step of "this tutorial is available on Github":https://github.com/piotrekinventica/tutorials/commits/history-jre-rest.
+
+  We recommend that you clone the repo locally:
+
+  bc[sh]. git clone https://github.com/ably/tutorials.git
+
+  Checkout the tutorial branch:
+
+  bc[sh]. git checkout history-jre-rest
```

Comment:
I also cloned `https://github.com/piotrekinventica/tutorials.git` for testing and it did not have that branch?